.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-3.1.3
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍